### ISSUE TITLE
TypeError [ERR_INVALID_ARG_TYPE] on some projects

### ISSUE THREAD

**Issue by heinolindstrom** [wallabyjs/public#2940]
### Issue description or question

I'm having problems running Wallaby.js on a project. I've updated the Core and jest runs normally from command line.

This is the error:

```
[Info]​ Started Wallaby.js Core v1.0.1235
​[Info]​ console.log: Jest:   ● Test suite failed to run
​[Info]​     TypeError [ERR_INVALID_ARG_TYPE]: The "newPath" argument must be of type string or an instance of Buffer or URL. Received undefined
​[Info]​       at writeFileSync (../../.vscode/extensions/wallabyjs.wallaby-vscode-1.0.327/wallaby538b7a/node_modules/write-file-atomic/lib/index.js:245:8)
​[Info]​       at s (../../.vscode/extensions/wallabyjs.wallaby-vscode-1.0.327/wallaby538b7a/runners/node/jest@24.8.0/initializer.js:15:2261)
​[Info]​       at TsJestTransformer.e.process (../../.vscode/extensions/wallabyjs.wallaby-vscode-1.0.327/wallaby538b7a/runners/node/jest@24.8.0/initializer.js:15:5058)
​[Info]​       at ScriptTransformer.transformSource (node_modules/@jest/transform/build/ScriptTransformer.js:612:31)
​[Info]​       at ScriptTransformer._transformAndBuildScript (node_modules/@jest/transform/build/ScriptTransformer.js:758:40)
​[Info]​       at ScriptTransformer.transform (node_modules/@jest/transform/build/ScriptTransformer.js:815:19)
```
### Wallaby diagnostics report

```
{
  editorVersion: '1.64.0',
  pluginVersion: '1.0.327',
  editorType: 'VSCode',
  osVersion: 'darwin 20.6.0',
  nodeVersion: 'v14.17.6',
  coreVersion: '1.0.1235',
  checksum: 'ZWM2MWE4NTk5ZjU1YjJmZDYyODEzNzM3OTUxZmY3NGUsMTY3OTA5NzYwMDAwMCww',
  config: {
    diagnostics: {
      jest: {
        config: {
          configs: [
            {
              automock: false,
              cache: true,
              cacheDirectory: '/private/var/folders/7l/ls3g869x7jlggjnr_h72s5kc0000gq/T/jest_dz',
              clearMocks: false,
              coveragePathIgnorePatterns: [ '/node_modules/' ],
              cwd: '<homeDir>/Projects/demo',
              dependencyExtractor: undefined,
              detectLeaks: false,
              detectOpenHandles: false,
              displayName: undefined,
              errorOnDeprecated: false,
              extensionsToTreatAsEsm: [],
              extraGlobals: [],
              filter: undefined,
              forceCoverageMatch: [],
              globalSetup: undefined,
              globalTeardown: undefined,
              globals: {},
              haste: { computeSha1: false, enableSymlinks: false, forceNodeFilesystemAPI: false, throwOnModuleCollision: false },
              injectGlobals: true,
              moduleDirectories: [ 'node_modules' ],
              moduleFileExtensions: [ 'js', 'jsx', 'ts', 'tsx', 'json', 'node' ],
              moduleLoader: undefined,
              moduleNameMapper: [],
              modulePathIgnorePatterns: [],
              modulePaths: undefined,
              name: 'f293f678a4c7a618e4347dd2ddd28fa2',
              prettierPath: 'prettier',
              resetMocks: false,
              resetModules: false,
              resolver: undefined,
              restoreMocks: false,
              rootDir: '<homeDir>/Projects/demo',
              roots: [ '<homeDir>/Projects/demo' ],
              runner: '<homeDir>/Projects/demo/node_modules/jest-runner/build/index.js',
              setupFiles: [],
              setupFilesAfterEnv: [ '<homeDir>/Projects/demo/test-helpers/setupTests.ts' ],
              skipFilter: false,
              skipNodeResolution: undefined,
              slowTestThreshold: 5,
              snapshotFormat: undefined,
              snapshotResolver: undefined,
              snapshotSerializers: [],
              testEnvironment: '<homeDir>/Projects/demo/node_modules/jest-environment-node/build/index.js',
              testEnvironmentOptions: {},
              testLocationInResults: false,
              testMatch: [ '**/__tests__/**/*.[jt]s?(x)', '**/?(*.)+(spec|test).[tj]s?(x)' ],
              testPathIgnorePatterns: [ 'node_modules', 'dist' ],
              testRegex: [],
              testRunner: '<homeDir>/Projects/demo/node_modules/jest-circus/runner.js',
              testURL: 'http://localhost',
              timers: 'real',
              transform: [ [ '^.+\\.tsx?$', '<homeDir>/Projects/demo/node_modules/ts-jest/dist/index.js', {} ] ],
              transformIgnorePatterns: [ '/node_modules/', '\\.pnp\\.[^\\/]+$' ],
              unmockedModulePathPatterns: undefined,
              watchPathIgnorePatterns: []
            }
          ],
          globalConfig: {
            bail: 0,
            changedFilesWithAncestor: false,
            changedSince: undefined,
            collectCoverage: false,
            collectCoverageFrom: [],
            collectCoverageOnlyFrom: undefined,
            coverageDirectory: '<homeDir>/Projects/demo/coverage',
            coverageProvider: 'babel',
            coverageReporters: [ 'json', 'text', 'lcov', 'clover' ],
            coverageThreshold: undefined,
            detectLeaks: false,
            detectOpenHandles: false,
            errorOnDeprecated: false,
            expand: false,
            filter: undefined,
            findRelatedTests: false,
            forceExit: false,
            globalSetup: undefined,
            globalTeardown: undefined,
            json: false,
            lastCommit: false,
            listTests: false,
            logHeapUsage: false,
            maxConcurrency: 5,
            maxWorkers: 7,
            noSCM: undefined,
            noStackTrace: false,
            nonFlagArgs: undefined,
            notify: false,
            notifyMode: 'failure-change',
            onlyChanged: false,
            onlyFailures: false,
            outputFile: undefined,
            passWithNoTests: false,
            projects: [],
            replname: undefined,
            reporters: undefined,
            rootDir: '<homeDir>/Projects/demo',
            runTestsByPath: false,
            silent: undefined,
            skipFilter: false,
            snapshotFormat: undefined,
            testFailureExitCode: 1,
            testNamePattern: undefined,
            testPathPattern: '',
            testResultsProcessor: undefined,
            testSequencer: '<homeDir>/Projects/demo/node_modules/@jest/test-sequencer/build/index.js',
            testTimeout: undefined,
            updateSnapshot: 'new',
            useStderr: false,
            verbose: undefined,
            watch: false,
            watchAll: false,
            watchPlugins: undefined,
            watchman: true
          },
          hasDeprecationWarnings: false,
          wallaby: {
            roots: [],
            watchPathIgnorePatterns: [
              '/node_modules/',
              '\\./dist/|\\./build/|\\./coverage/|\\./git/|/\\..+/',
              '/private/var/folders/7l/ls3g869x7jlggjnr_h72s5kc0000gq/T/jest_dz',
              'node_modules',
              'dist',
              '\\./coverage'
            ],
            testPathIgnorePatterns: [
              '/node_modules/',
              '\\./dist/|\\./build/|\\./coverage/|\\./git/|/\\..+/',
              '/private/var/folders/7l/ls3g869x7jlggjnr_h72s5kc0000gq/T/jest_dz',
              'node_modules',
              'dist',
              '\\./coverage'
            ],
            testMatch: [ '**/__tests__/**/*.[jt]s?(x)', '**/?(*.)+(spec|test).[tj]s?(x)' ],
            testRegex: []
          }
        }
      }
    },
    testFramework: { version: 'jest@24.8.0', configurator: 'jest@24.8.0', reporter: 'jest@24.8.0', starter: 'jest@24.8.0', autoDetected: true },
    filesWithCoverageCalculated: [],
    filesWithNoCoverageCalculated: [],
    globalSetup: false,
    micromatch: true,
    files: [
      { pattern: '/node_modules/', regexp: /\/node_modules\//, ignore: true, trigger: true, load: true },
      { pattern: '\\./dist/|\\./build/|\\./coverage/|\\./git/|/\\..+/', regexp: /\.\/dist\/|\.\/build\/|\.\/coverage\/|\.\/git\/|\/\..+\//, ignore: true, trigger: true, load: true },
      {
        pattern: '/private/var/folders/7l/ls3g869x7jlggjnr_h72s5kc0000gq/T/jest_dz',
        regexp: /\/private\/var\/folders\/7l\/ls3g869x7jlggjnr_h72s5kc0000gq\/T\/jest_dz/,
        ignore: true,
        trigger: true,
        load: true
      },
      { pattern: 'node_modules', regexp: /node_modules/, ignore: true, trigger: true, load: true },
      { pattern: 'dist', regexp: /dist/, ignore: true, trigger: true, load: true },
      { pattern: '\\./coverage', regexp: /\.\/coverage/, ignore: true, trigger: true, load: true },
      { pattern: '**/**', ignore: false, trigger: true, load: true, order: 1 },
      { pattern: '**/__tests__/**/*.[jt]s?(x)', ignore: true, trigger: true, load: true },
      { pattern: '**/?(*.)+(spec|test).[tj]s?(x)', ignore: true, trigger: true, load: true }
    ],
    tests: [
      { pattern: '/node_modules/', regexp: /\/node_modules\//, ignore: true, trigger: true, load: true, test: true },
      {
        pattern: '\\./dist/|\\./build/|\\./coverage/|\\./git/|/\\..+/',
        regexp: /\.\/dist\/|\.\/build\/|\.\/coverage\/|\.\/git\/|\/\..+\//,
        ignore: true,
        trigger: true,
        load: true,
        test: true
      },
      {
        pattern: '/private/var/folders/7l/ls3g869x7jlggjnr_h72s5kc0000gq/T/jest_dz',
        regexp: /\/private\/var\/folders\/7l\/ls3g869x7jlggjnr_h72s5kc0000gq\/T\/jest_dz/,
        ignore: true,
        trigger: true,
        load: true,
        test: true
      },
      { pattern: 'node_modules', regexp: /node_modules/, ignore: true, trigger: true, load: true, test: true },
      { pattern: 'dist', regexp: /dist/, ignore: true, trigger: true, load: true, test: true },
      { pattern: '\\./coverage', regexp: /\.\/coverage/, ignore: true, trigger: true, load: true, test: true },
      { pattern: '**/__tests__/**/*.[jt]s?(x)', ignore: false, trigger: true, load: true, test: true, order: 2 },
      { pattern: '**/?(*.)+(spec|test).[tj]s?(x)', ignore: false, trigger: true, load: true, test: true, order: 3 }
    ],
    runAllTestsInAffectedTestFile: false,
    updateNoMoreThanOneSnapshotPerTestFileRun: false,
    addModifiedTestFileToExclusiveTestRun: true,
    compilers: {},
    preprocessors: {},
    maxConsoleMessagesPerTest: 100,
    autoConsoleLog: true,
    delays: { run: 0, edit: 100, update: 0 },
    workers: { initial: 0, regular: 0, recycle: false },
    teardown: undefined,
    hints: {
      ignoreCoverage: '__REGEXP /ignore coverage|istanbul ignore/',
      ignoreCoverageForFile: '__REGEXP /ignore file coverage/',
      commentAutoLog: '?',
      testFileSelection: { include: '__REGEXP /file\\.only/', exclude: '__REGEXP /file\\.skip/' }
    },
    automaticTestFileSelection: true,
    runSelectedTestsOnly: false,
    mapConsoleMessagesStackTrace: false,
    extensions: {},
    env: {
      type: 'node',
      params: {},
      runner: '<homeDir>/.nvm/versions/node/v14.17.6/bin/node',
      viewportSize: { width: 800, height: 600 },
      options: { width: 800, height: 600 },
      bundle: true
    },
    reportUnhandledPromises: true,
    slowTestThreshold: 75,
    lowCoverageThreshold: 80,
    loose: true,
    configCode: 'auto.detect#-655885702'
  },
  packageJSON: {
    dependencies: {
      '@azure/identity': '^2.0.4',
      '@azure/keyvault-certificates': '^4.3.0'
    },
    devDependencies: {
      '@azure/event-hubs': '^5.6.0',
      '@azure/functions': '^1.0.1-beta1',
      '@types/date-fns': '^2.6.0',
      '@types/jest': '^27.0.2',
      '@types/node': '^16.10.1',
    }
  },
  fs: { numberOfFiles: 176 },
  debug: [
    '2022-03-05T07:21:54.925Z angular/cli config Angular CLI not found.\n',
    '2022-03-05T07:21:55.075Z jest/config Detected Jest.\n',
    '2022-03-05T07:21:55.076Z jest/config Configured Jest.\n',
    '2022-03-05T07:21:55.076Z project Wallaby Node version: v14.17.6\n',
    '2022-03-05T07:21:55.076Z project Wallaby config: <homeDir>/Projects/demo/auto.detect\n',
    '2022-03-05T07:21:55.138Z project File cache: <homeDir>/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.327/projects/02912919b3d9d97f\n',
    '2022-03-05T07:21:55.142Z uiService Listening port 51236\n',
    '2022-03-05T07:21:55.153Z workers Parallelism for initial run: 6, for regular run: 3\n',
    '2022-03-05T07:21:55.153Z workers Starting run worker instance #0\n',
    '2022-03-05T07:21:55.153Z workers Starting run worker instance #1\n',
    '2022-03-05T07:21:55.153Z workers Starting run worker instance #2\n',
    '2022-03-05T07:21:55.153Z workers Starting run worker instance #3\n',
    '2022-03-05T07:21:55.153Z workers Starting run worker instance #4\n',
    '2022-03-05T07:21:55.153Z workers Starting run worker instance #5\n',
    '2022-03-05T07:21:55.154Z workers Web server is listening at 57814\n',
    '2022-03-05T07:21:55.169Z project Stopping process pool\n',
    '2022-03-05T07:21:55.169Z project File cache is up-to-date, starting full test run\n',
    '2022-03-05T07:21:55.170Z project Test run started; run priority: 3\n',
    '2022-03-05T07:21:55.170Z project Running all tests\n',
    '2022-03-05T07:21:55.172Z workers Starting test run, priority: 3\n',
    '2022-03-05T07:21:55.172Z workers Distributing tests between 6 workers\n',
    '2022-03-05T07:21:55.172Z workers Running tests in parallel\n',
    '2022-03-05T07:21:55.172Z nodeRunner Starting sandbox [worker #0, session #95ai3]\n',
    '2022-03-05T07:21:55.172Z nodeRunner Starting sandbox [worker #1, session #uzpvc]\n',
    '2022-03-05T07:21:55.172Z nodeRunner Starting sandbox [worker #2, session #wgfzn]\n',
    '2022-03-05T07:21:55.172Z nodeRunner Starting sandbox [worker #3, session #azr1z]\n',
    '2022-03-05T07:21:55.172Z nodeRunner Starting sandbox [worker #4, session #afjlx]\n',
    '2022-03-05T07:21:55.172Z nodeRunner Starting sandbox [worker #5, session #a00lp]\n',
    '2022-03-05T07:21:55.173Z nodeRunner Preparing sandbox [worker #0, session #95ai3]\n',
    '2022-03-05T07:21:55.173Z nodeRunner Preparing sandbox [worker #1, session #uzpvc]\n',
    '2022-03-05T07:21:55.173Z nodeRunner Preparing sandbox [worker #2, session #wgfzn]\n',
    '2022-03-05T07:21:55.173Z nodeRunner Preparing sandbox [worker #3, session #azr1z]\n',
    '2022-03-05T07:21:55.173Z nodeRunner Preparing sandbox [worker #4, session #afjlx]\n',
    '2022-03-05T07:21:55.173Z nodeRunner Preparing sandbox [worker #5, session #a00lp]\n',
    '2022-03-05T07:21:55.316Z workers Started run worker instance (delayed) #0\n',
    '2022-03-05T07:21:55.316Z nodeRunner Prepared sandbox [worker #0, session #95ai3]\n',
    '2022-03-05T07:21:55.317Z workers [worker #0, session #95ai3] Running tests in sandbox\n',
    '2022-03-05T07:21:55.396Z workers Started run worker instance (delayed) #1\n',
    '2022-03-05T07:21:55.396Z nodeRunner Prepared sandbox [worker #1, session #uzpvc]\n',
    '2022-03-05T07:21:55.396Z workers [worker #1, session #uzpvc] Running tests in sandbox\n',
    '2022-03-05T07:21:55.484Z workers Started run worker instance (delayed) #3\n',
    '2022-03-05T07:21:55.484Z nodeRunner Prepared sandbox [worker #3, session #azr1z]\n',
    '2022-03-05T07:21:55.484Z workers [worker #3, session #azr1z] Running tests in sandbox\n',
    '2022-03-05T07:21:55.579Z workers Started run worker instance (delayed) #2\n',
    '2022-03-05T07:21:55.579Z nodeRunner Prepared sandbox [worker #2, session #wgfzn]\n',
    '2022-03-05T07:21:55.579Z workers [worker #2, session #wgfzn] Running tests in sandbox\n',
    '2022-03-05T07:21:55.667Z workers Started run worker instance (delayed) #4\n',
    '2022-03-05T07:21:55.667Z nodeRunner Prepared sandbox [worker #4, session #afjlx]\n',
    '2022-03-05T07:21:55.667Z workers [worker #4, session #afjlx] Running tests in sandbox\n',
    '2022-03-05T07:21:55.750Z workers Started run worker instance (delayed) #5\n',
    '2022-03-05T07:21:55.751Z nodeRunner Prepared sandbox [worker #5, session #a00lp]\n',
    '2022-03-05T07:21:55.751Z workers [worker #5, session #a00lp] Running tests in sandbox\n',
    '2022-03-05T07:22:00.977Z project Test run is not cancelled because cancel requester does not have enough priority to cancel the run\n',
    '2022-03-05T07:22:01.032Z workers Scheduling Jest Test Run (uzpvc): 2022-03-05T07:21:57.642Z\n',
    '2022-03-05T07:22:01.034Z workers Sandbox (active) [uzpvc] error: The "newPath" argument must be of type string or an instance of Buffer or URL. Received undefined\n',
    '2022-03-05T07:22:01.050Z workers Scheduling Jest Test Run (azr1z): 2022-03-05T07:21:57.642Z\n',
    '2022-03-05T07:22:01.051Z workers Sandbox (active) [azr1z] error: The "newPath" argument must be of type string or an instance of Buffer or URL. Received undefined\n',
    '2022-03-05T07:22:01.054Z workers [uzpvc] Run 0 test(s), skipped 0 test(s)\n',
    '2022-03-05T07:22:01.055Z workers Jest Test Run Complete (uzpvc): 2022-03-05T07:22:01.030Z\n',
    '2022-03-05T07:22:01.056Z workers [uzpvc] Sandbox is responsive, closing it\n',
    '2022-03-05T07:22:01.057Z workers Failed to map the stack to user code, entry message: The "newPath" argument must be of type string or an instance of Buffer or URL. Received undefined, stack: TypeError [ERR_INVALID_ARG_TYPE]: The "newPath" argument must be of type string or an instance of Buffer or URL. Received undefined\n' +
      '    at Object.renameSync (fs.js:790:13)\n' +
      '    at writeFileSync (<homeDir>/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.327/wallaby538b7a/node_modules/write-file-atomic/lib/index.js:245:8)\n' +
      '    at s (<homeDir>/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.327/wallaby538b7a/runners/node/jest@24.8.0/initializer.js:15:2261)\n' +
      '    at TsJestTransformer.e.process (<homeDir>/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.327/wallaby538b7a/runners/node/jest@24.8.0/initializer.js:15:5058)\n' +
      '    at ScriptTransformer.transformSource (<homeDir>/Projects/demo/node_modules/@jest/transform/build/ScriptTransformer.js:612:31)\n' +
      '    at ScriptTransformer._transformAndBuildScript (<homeDir>/Projects/demo/node_modules/@jest/transform/build/ScriptTransformer.js:758:40)\n' +
      '    at ScriptTransformer.transform (/Us\n',
    '2022-03-05T07:22:01.070Z fs File deleted: undefined.1368128529\n',
    '2022-03-05T07:22:01.081Z project Test run is not cancelled because cancel requester does not have enough priority to cancel the run\n',
    "2022-03-05T07:22:01.087Z fs TypeError: Cannot read property 'id' of undefined\n" +
      '    at _0x139df4.<computed>._updateProjectFile (<homeDir>/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.327/wallaby538b7a/node_modules/ws/lib/Sender.fallbacks.js:1:203645)\n' +
      '    at _0x301cba.wrapper (<homeDir>/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.327/wallaby538b7a/node_modules/lodash/index.js:3141:56)\n' +
      '    at _0x301cba.emit (events.js:400:28)\n' +
      '    at <homeDir>/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.327/wallaby538b7a/node_modules/ws/lib/Sender.fallbacks.js:1:381020\n' +
      '    at processTicksAndRejections (internal/process/task_queues.js:95:5)\n',
    '2022-03-05T07:22:01.087Z project Test run is not cancelled because cancel requester does not have enough priority to cancel the run\n',
    '2022-03-05T07:22:01.088Z workers Scheduling Jest Test Run (afjlx): 2022-03-05T07:21:57.643Z\n',
    '2022-03-05T07:22:01.099Z workers Sandbox (active) [afjlx] error: The "newPath" argument must be of type string or an instance of Buffer or URL. Received undefined\n',
    '2022-03-05T07:22:01.100Z workers [azr1z] Run 0 test(s), skipped 0 test(s)\n',
    '2022-03-05T07:22:01.101Z workers Jest Test Run Complete (azr1z): 2022-03-05T07:22:01.049Z\n',
    '2022-03-05T07:22:01.101Z workers [azr1z] Sandbox is responsive, closing it\n',
    '2022-03-05T07:22:01.101Z workers Failed to map the stack to user code, entry message: The "newPath" argument must be of type string or an instance of Buffer or URL. Received undefined, stack: TypeError [ERR_INVALID_ARG_TYPE]: The "newPath" argument must be of type string or an instance of Buffer or URL. Received undefined\n' +
      '    at Object.renameSync (fs.js:790:13)\n' +
      '    at writeFileSync (<homeDir>/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.327/wallaby538b7a/node_modules/write-file-atomic/lib/index.js:245:8)\n' +
      '    at s (<homeDir>/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.327/wallaby538b7a/runners/node/jest@24.8.0/initializer.js:15:2261)\n' +
      '    at TsJestTransformer.e.process (<homeDir>/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.327/wallaby538b7a/runners/node/jest@24.8.0/initializer.js:15:5058)\n' +
      '    at ScriptTransformer.transformSource (<homeDir>/Projects/demo/node_modules/@jest/transform/build/ScriptTransformer.js:612:31)\n' +
      '    at ScriptTransformer._transformAndBuildScript (<homeDir>/Projects/demo/node_modules/@jest/transform/build/ScriptTransformer.js:758:40)\n' +
      '    at ScriptTransformer.transform (/Us\n',
    '2022-03-05T07:22:01.107Z workers Scheduling Jest Test Run (wgfzn): 2022-03-05T07:21:57.643Z\n',
    '2022-03-05T07:22:01.108Z workers Scheduling Jest Test Run (95ai3): 2022-03-05T07:21:57.643Z\n',
    '2022-03-05T07:22:01.110Z workers Sandbox (active) [wgfzn] error: The "newPath" argument must be of type string or an instance of Buffer or URL. Received undefined\n',
    '2022-03-05T07:22:01.112Z workers Sandbox (active) [95ai3] error: The "newPath" argument must be of type string or an instance of Buffer or URL. Received undefined\n',
    '2022-03-05T07:22:01.128Z workers [afjlx] Run 0 test(s), skipped 0 test(s)\n',
    '2022-03-05T07:22:01.129Z workers [wgfzn] Run 0 test(s), skipped 0 test(s)\n',
    '2022-03-05T07:22:01.129Z workers Jest Test Run Complete (afjlx): 2022-03-05T07:22:01.082Z\n',
    '2022-03-05T07:22:01.130Z workers [afjlx] Sandbox is responsive, closing it\n',
    '2022-03-05T07:22:01.130Z workers Failed to map the stack to user code, entry message: The "newPath" argument must be of type string or an instance of Buffer or URL. Received undefined, stack: TypeError [ERR_INVALID_ARG_TYPE]: The "newPath" argument must be of type string or an instance of Buffer or URL. Received undefined\n' +
      '    at Object.renameSync (fs.js:790:13)\n' +
      '    at writeFileSync (<homeDir>/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.327/wallaby538b7a/node_modules/write-file-atomic/lib/index.js:245:8)\n' +
      '    at s (<homeDir>/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.327/wallaby538b7a/runners/node/jest@24.8.0/initializer.js:15:2261)\n' +
      '    at TsJestTransformer.e.process (<homeDir>/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.327/wallaby538b7a/runners/node/jest@24.8.0/initializer.js:15:5058)\n' +
      '    at ScriptTransformer.transformSource (<homeDir>/Projects/demo/node_modules/@jest/transform/build/ScriptTransformer.js:612:31)\n' +
      '    at ScriptTransformer._transformAndBuildScript (<homeDir>/Projects/demo/node_modules/@jest/transform/build/ScriptTransformer.js:758:40)\n' +
      '    at ScriptTransformer.transform (/Us\n',
    '2022-03-05T07:22:01.130Z workers Jest Test Run Complete (wgfzn): 2022-03-05T07:22:01.106Z\n',
    '2022-03-05T07:22:01.130Z workers [wgfzn] Sandbox is responsive, closing it\n',
    '2022-03-05T07:22:01.131Z workers Failed to map the stack to user code, entry message: The "newPath" argument must be of type string or an instance of Buffer or URL. Received undefined, stack: TypeError [ERR_INVALID_ARG_TYPE]: The "newPath" argument must be of type string or an instance of Buffer or URL. Received undefined\n' +
      '    at Object.renameSync (fs.js:790:13)\n' +
      '    at writeFileSync (<homeDir>/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.327/wallaby538b7a/node_modules/write-file-atomic/lib/index.js:245:8)\n' +
      '    at s (<homeDir>/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.327/wallaby538b7a/runners/node/jest@24.8.0/initializer.js:15:2261)\n' +
      '    at TsJestTransformer.e.process (<homeDir>/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.327/wallaby538b7a/runners/node/jest@24.8.0/initializer.js:15:5058)\n' +
      '    at ScriptTransformer.transformSource (<homeDir>/Projects/demo/node_modules/@jest/transform/build/ScriptTransformer.js:612:31)\n' +
      '    at ScriptTransformer._transformAndBuildScript (<homeDir>/Projects/demo/node_modules/@jest/transform/build/ScriptTransformer.js:758:40)\n' +
      '    at ScriptTransformer.transform (/Us\n',
    '2022-03-05T07:22:01.131Z workers [95ai3] Run 0 test(s), skipped 0 test(s)\n',
    '2022-03-05T07:22:01.131Z workers Jest Test Run Complete (95ai3): 2022-03-05T07:22:01.103Z\n',
    '2022-03-05T07:22:01.132Z workers [95ai3] Sandbox is responsive, closing it\n',
    '2022-03-05T07:22:01.133Z workers Failed to map the stack to user code, entry message: The "newPath" argument must be of type string or an instance of Buffer or URL. Received undefined, stack: TypeError [ERR_INVALID_ARG_TYPE]: The "newPath" argument must be of type string or an instance of Buffer or URL. Received undefined\n' +
      '    at Object.renameSync (fs.js:790:13)\n' +
      '    at writeFileSync (<homeDir>/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.327/wallaby538b7a/node_modules/write-file-atomic/lib/index.js:245:8)\n' +
      '    at s (<homeDir>/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.327/wallaby538b7a/runners/node/jest@24.8.0/initializer.js:15:2261)\n' +
      '    at TsJestTransformer.e.process (<homeDir>/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.327/wallaby538b7a/runners/node/jest@24.8.0/initializer.js:15:5058)\n' +
      '    at ScriptTransformer.transformSource (<homeDir>/Projects/demo/node_modules/@jest/transform/build/ScriptTransformer.js:612:31)\n' +
      '    at ScriptTransformer._transformAndBuildScript (<homeDir>/Projects/demo/node_modules/@jest/transform/build/ScriptTransformer.js:758:40)\n' +
      '    at ScriptTransformer.transform (/Us\n',
    '2022-03-05T07:22:01.156Z fs File deleted: undefined.2285785233\n',
    '2022-03-05T07:22:01.156Z project Test run is not cancelled because cancel requester does not have enough priority to cancel the run\n',
    '2022-03-05T07:22:01.157Z fs File deleted: undefined.3194006135\n',
    '2022-03-05T07:22:01.157Z project Test run is not cancelled because cancel requester does not have enough priority to cancel the run\n',
    '2022-03-05T07:22:01.170Z workers Scheduling Jest Test Run (a00lp): 2022-03-05T07:21:57.661Z\n',
    '2022-03-05T07:22:01.172Z workers Sandbox (active) [a00lp] error: The "newPath" argument must be of type string or an instance of Buffer or URL. Received undefined\n',
    '2022-03-05T07:22:01.179Z workers [a00lp] Run 0 test(s), skipped 0 test(s)\n',
    '2022-03-05T07:22:01.180Z workers Jest Test Run Complete (a00lp): 2022-03-05T07:22:01.169Z\n',
    '2022-03-05T07:22:01.181Z workers [a00lp] Sandbox is responsive, closing it\n',
    '2022-03-05T07:22:01.181Z workers Failed to map the stack to user code, entry message: The "newPath" argument must be of type string or an instance of Buffer or URL. Received undefined, stack: TypeError [ERR_INVALID_ARG_TYPE]: The "newPath" argument must be of type string or an instance of Buffer or URL. Received undefined\n' +
      '    at Object.renameSync (fs.js:790:13)\n' +
      '    at writeFileSync (<homeDir>/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.327/wallaby538b7a/node_modules/write-file-atomic/lib/index.js:245:8)\n' +
      '    at s (<homeDir>/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.327/wallaby538b7a/runners/node/jest@24.8.0/initializer.js:15:2261)\n' +
      '    at TsJestTransformer.e.process (<homeDir>/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.327/wallaby538b7a/runners/node/jest@24.8.0/initializer.js:15:5058)\n' +
      '    at ScriptTransformer.transformSource (<homeDir>/Projects/demo/node_modules/@jest/transform/build/ScriptTransformer.js:612:31)\n' +
      '    at ScriptTransformer._transformAndBuildScript (<homeDir>/Projects/demo/node_modules/@jest/transform/build/ScriptTransformer.js:758:40)\n' +
      '    at ScriptTransformer.transform (/Us\n',
    '2022-03-05T07:22:01.181Z workers Merging parallel test run results\n',
    '2022-03-05T07:22:01.186Z project Test run finished\n',
    '2022-03-05T07:22:01.186Z project Processed console.log entries\n',
    '2022-03-05T07:22:01.186Z project Processed loading sequences\n',
    '2022-03-05T07:22:01.186Z project Processed executed tests\n',
    '2022-03-05T07:22:01.188Z project Processed code coverage\n',
    '2022-03-05T07:22:01.205Z project Test run result processed and sent to IDE\n',
    '2022-03-05T07:22:01.210Z project Changes did not trigger any tests\n'
  ]
}
```

**Comment by ArtemGovorov** [comment]
Thanks fo the report, the issue is fixed and the fix is published in [the latest core v1.0.1236](https://wallabyjs.com/docs/intro/troubleshooting.html#forcing-wallabyjs-core-update).

**Comment by heinolindstrom** [comment]
Thanks! 👍 Weird that I didn't get the latest version this morning.
